mautic/mautic
# Fix absolute url of preview email

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.3
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:

Email preview link was not absolute url. This PR fixed it



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create email
3. See preview link is in format /email/preview/xxx, expect https://mauticurl/email/preview/xxx
4. This PR fixed it

![image](https://user-images.githubusercontent.com/462477/97103988-6802c800-16b0-11eb-8ce6-76655604436a.png)
